atom/renderer/atom_render_frame_observer.h

```diff
@@ -21,9 +21,15 @@ namespace atom {
 
 enum World {
   MAIN_WORLD = 0,
+
   // Use a high number far away from 0 to not collide with any other world
   // IDs created internally by Chrome.
-  ISOLATED_WORLD = 999
+  ISOLATED_WORLD = 999,
+
+  // Numbers for isolated worlds for extensions are set in
+  // lib/renderer/content-script-injector.js, and are greater than or equal to
+  // this number.
+  ISOLATED_WORLD_EXTENSIONS = 1000,
```

Comment:
This will have to be explicitly documented in the `executeJavaScriptInIsolatedWorld` API docs that these numbers are "reserved" / "used" by chrome extensions

Reply:
Good point. Since users may have use cases for a large range, perhaps the extensions range should be moved up as well (`1000` => `100000`?).

Reply:
I've defined the range between `[1 << 20, 1 << 29)` with more info in the commit message of 8d020154d4d7ebb2deb13b44ff0555baa2b39a13